.pre-commit-config-cpp.yaml

```diff
@@ -0,0 +1,26 @@
+# This file contains pre-commit hooks for checking the C++ code. Since these
+# hooks require tools that may not be trivial to install (unlike the Python
+# hooks like flake8), we keep these in a separate file to be applied manually.
+repos:
+  - repo: local
+    hooks:
+      - id: clang-format
+        name: clang-format
+        entry: clang-format -i
+        language: system
+        types: [c++]
+        # Exclude specific files.
+        exclude: |
+          (?x)(
+            ^CMake/|
+            ^hoomd/extern/|
+            ^hoomd/hpmc/UpdaterClusters\.h|
+            ^hoomd/hpmc/IntegratorHPMCMono\.h|
+            ^doc/dev/
+          )
+        verbose: true
+        # Unclear if --style=file is necessary, it seems like clang-format uses
+        # .clang-format if found by default, at least for newer versions. I'll
+        # specify this here just in case.
+        args:
+        - --style=file
```

Comment:
This option shouldn't be necessary. It is the default.

Reply:
Thanks, I wasn't sure about this. Vyas configured this for freud and I left this argument with the comment. I'll remove it now.

Reply:
The clang-format version that we were originally working with in freud was 6.0 I think, and it wouldn't search for `.clang-format` without passing this option explicitly. Agreed that this can be removed now.